src/fn/edit-contacts.js

```diff
@@ -21,9 +21,26 @@ const overwriteFileCheck = (doc, args, overwriteAllFiles) => {
 };
 
 const saveJsonDoc = (doc, args) => {
-  const writeFile = (writeDoc) => fs.write(jsonDocPath(args.docDirectoryPath, writeDoc._id), safeStringify(writeDoc) + '\n');
-  
-  return writeFile(doc);
+  return fs.write(jsonDocPath(args.docDirectoryPath, doc._id), safeStringify(doc) + '\n');
+};
+
+const writeDocs = (docs, args, overwriteAllFiles) => {
```

Comment:
I think `overwriteAllFiles` should just be initialized here, and not passed as a parameter. Is it used outside the scope of this function?

Reply:
You're right. No need to pass it as a parameter.